controllers/toplevel/index.js

```diff
@@ -88,13 +107,17 @@ module.exports = ({ router, pages }) => {
 
                 res.send({
                     status: surveyToShow ? 'success' : 'error',
+                    lang: {
```

Comment:
Including any global translations here means we can make the whole survey component client-side. Avoids the half-vue / half-nunjucks interpolation issue.